config-solaris.md

```diff
@@ -90,7 +90,7 @@ Mapped to `allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
 * **`configureAllowedAddress`** *(string, OPTIONAL)* If configureAllowedAddress is set to true, the addresses specified by allowedAddress are automatically configured on the interface each time the container starts.
 When it is set to false, the allowedAddress will not be configured on container start.
 Mapped to `configure-allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
-* **`defrouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
+* **`defaultRouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
```

Comment:
hey @wking and @jhowardmsft.  i'll be taking over responsibility for the solaris components from @anuthan.  i've reviewed our use of camel case and i don't actually want to make any changes to it.  in most cases our property names mirror what's found in zonecfg ("defrouter" being an example), and camel case was only used for property names that had a hyphen in them.  for example, the zonecfg "allowed-address" property is mapped to "allowedAddress".  hopefully that sounds reasonable?

that said, some of our docs.oracle.com links are broken so i'll file and issue for that and get it fixed.